### ISSUE TITLE
 Gateway-proxy can't start with readiness probe enabled

### ISSUE THREAD

**Issue by mhmtsvr** [solo-io/gloo#9092]
### Gloo Edge Product

Open Source

### Gloo Edge Version

v1.16.2

### Kubernetes Version

v1.27

### Describe the bug

Gateway-proxy can't start when readiness probe is enabled, because wget is not found in $PATH. The same issue https://github.com/solo-io/gloo/issues/8749 occurs in v1.16.2 again.

### Expected Behavior

The readiness probe should succeed, and the pods should transition to a ready state.

### Steps to reproduce the bug

Install gloo with values:

```
gloo:
  gatewayProxies:
    gatewayProxy:
      podTemplate:
        probes: true
```

Check kubectl events of the gateway-proxy pod and you'll see:

```
12s (x13 over 2m1s)     Warning   Unhealthy               Pod/gateway-proxy-64d4cb6467-mzcr2                             (combined from similar events): Readiness probe errored: rpc error: code = Unknown desc = failed to exec in container: failed to start exec "a32d5804a6c0de314af93b2a2e82ee06d26481d8603c29240bb9264d6781b94c": OCI runtime exec failed: exec failed: unable to start container process: exec: "wget": executable file not found in $PATH: unknown
```

### Additional Environment Detail

As per https://github.com/solo-io/gloo/issues/8749, a work around might be to use a custom readiness probe, but it'd be awesome if this gets fixed soon.

**Comment by mhmtsvr** [comment]
Just an FYI, the bug does still exist in `1.16.3`.

**Comment by nfuden** [comment]
Thanks! For raising this I think I see the issue we missed a forward port in our open source repo.

**Comment by nfuden** [comment]
Fixed in 1.16.2
